Sprint-2/debug/1.js

```diff
@@ -1,8 +1,12 @@
 // Predict and explain first...
 
 function sum(a, b) {
-  return;
-  a + b;
+  return a + b;
 }
 
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
+
+
+// the function will return undefined, because when it is called in last line, it starts with return statement followed by semicolon which causes the function to end without execution a + b statement.
```

Comment:
```suggestion
// the function will return undefined, because the return statement causes the function to end without execution a + b statement. Nothing is executed after the return statement line. 
```

Reply:
Much clear now, thank you.